Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyAssetCtrl.cpp

```diff
@@ -1072,10 +1072,10 @@ namespace AzToolsFramework
             RefreshAutocompleter();
         }
 
-        // When focus is lost, clear the field if necessary
+        // When focus is lost, revert to the selected asset
         if (!focus && m_incompleteFilename)
         {
-            HandleFieldClear();
+            SetSelectedAssetID(GetCurrentAssetID());
```

Comment:
question: As this looks like a behavior change it might be something we want UX to signoff on, would it be possible to include a screenshot or video of the before/after behavior please and/or description of exactly what the effect of this change is? Thanks!

Reply:
This change has been implemented in the latest version of development so it should no longer be relevant. But the PR was submitted before that.

Reply:
Aaah I see okay, thanks for the context, I'll let @AMZN-AlexOteiza chime in for his take. Cheers!